analytics-listeners/src/main/java/org/exoplatform/analytics/listener/chat/ChatSentMessageListener.java

```diff
@@ -0,0 +1,66 @@
+package org.exoplatform.analytics.listener.chat;
+
+import org.exoplatform.analytics.model.StatisticData;
+import org.exoplatform.analytics.utils.AnalyticsUtils;
+import org.exoplatform.services.listener.Asynchronous;
+import org.exoplatform.services.listener.Event;
+import org.exoplatform.services.listener.Listener;
+import org.exoplatform.social.core.space.model.Space;
+import org.exoplatform.social.core.space.spi.SpaceService;
+import org.json.JSONArray;
+import org.json.JSONException;
+
+import java.util.HashMap;
+
+@Asynchronous
+public class ChatSentMessageListener extends Listener<String, HashMap<String,String>> {
```

Comment:
Not needed, see comment on [Chat PR](https://github.com/exoplatform/chat-application/pull/204)